dangerfile.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable */
```

Comment:
Is there a reason ESLint is disabled for this file? (I also noticed tabs instead of spaces.)

You can always just use `/* global danger, warn, fail */` if you just need the globals.